### DISCUSSION TITLE
How to lock the reference window after the keymap trigger?

### DISCUSSION THREAD

**Discussion by OriLiMu** [LazyVim/LazyVim#3725] [Q&A] [answered]
Here is my keymap for `lsp_references`
`keymap("n", "<A-S-f>", "<cmd>Trouble lsp_references<CR>", { silent = true })`

After I use the keymap to trigger it on the variable `a` for example. 
But it will auto change the lsp_references result when I move the cursor to another variable or function name. 
When I move cursor on `b` the result wil be references about `b`
How to make it locked?

**Comment by dpetka2001** [comment, marked as answer]
https://github.com/folke/trouble.nvim/issues/430#issuecomment-2111716023
